.github/workflows/met-etl-cd.yml

```diff
@@ -0,0 +1,64 @@
+name: MET ETL DEV CD
+
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "met-etl/**"
+      - "met-api/**"
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment (dev/test/prod)"
+        required: true
+        default: "dev"
+
+defaults:
+  run:
+    shell: bash
+    working-directory: ./met-etl
+
+env:
+  APP_NAME: "dagster-etl"
+  DEPLOYMENT_NAME: "dagster-dagster-user-deployments-k8s-example-user-code-1"
```

Comment:
@saravanpa-aot once I understand dagster deployment better we can have a better name for this